### ISSUE TITLE
Duplicate content between section 9.6-9.6.2 and 10.3-10.3.2

### ISSUE THREAD

**Issue by V0XNIHILI** [harvard-edge/cs249r_book#96]
Comparing the content in these two sections, it seems that they are 99% identical. Probably best to keep either one of them?

**Comment by profvjreddi** [comment]
Good catch, yea we drafted 10 after setting up 9. My hope was that we make 9 a general overview before diving into the details. But I agree that it needs to be improved.
